samples/compose-jvm-app/common/src/desktopMain/kotlin/me/amikhailov/common/Main.kt

```diff
@@ -11,11 +11,11 @@ import androidx.compose.ui.unit.DpSize
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.window.WindowState
 import androidx.compose.ui.window.singleWindowApplication
+import com.icerockdev.app.MRdesktopMain
 
 fun main() {
-
     singleWindowApplication(
-        title = "moko-resources",
+        title = MRdesktopMain.strings.title.localized(),
```

Comment:
in this sample we have also commonMain resources too? 
if not - our desktopMain resources should be in MR, not MRdesktopMain.

Reply:
We had problems with jvm target and for check correct realization of plugin, i need add sample with JVM resources. I think simple solution - add Desktop resources and Common in current sample

Reply:
for title in desktop resources value changed on moko-resources

Reply:
checked, all ok